lib/core/sessions.js

```diff
@@ -364,6 +376,13 @@ function attemptTransaction(session, startTime, fn, options) {
           return attemptTransaction(session, startTime, fn, options);
         }
 
+        if (isMaxTimeMSExpiredError(err)) {
+          if (err.errorLabels == null) {
```

Comment:
this would play into my comment on #2102 where if we could ensure that errorLabels always exists, we could get rid of this check.